README.md

```diff
@@ -361,6 +361,17 @@ CUDA_VISIBLE_DEVICES=0,1 torchrun --nproc_per_node 2 scripts/inference.py config
 
 :warning: **LIMITATION**: The sequence parallelism is not supported for gradio deployment. For now, the sequence parallelism is only supported when the dimension can be divided by the number of GPUs. Thus, it may fail for some cases. We tested 4 GPUs for 720p and 2 GPUs for 480p.
 
+### Separate Inference 720p video with 24G VRAM
+```bash
+# text to video
+./scripts/separate_inference.sh 4s 720p "9:16" "a beautiful waterfall"
+```
+
+```bash
+# image to video
+./scripts/separate_inference.sh 4s 720p "9:16" "a beautiful waterfall. {\"reference_path\": \"path2reference.png\",\"mask_strategy\": \"0\"}""
+```
+
```

Comment:
I am aware of your motivation, but can you add more doc to tell other users why, when and how to run inference separately so that they can feel more guided?

Reply:
Yes, I'll add that later.